### PR TITLE
Add doc support for interleaving search in Graph Search API

### PR DESCRIPTION
**Pull request by yiwenwang-ms**

### CODE REVIEW

File: concepts/search-concept-interleaving.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+## Known limitations
+
+- Customized Sort is not supported in interleaving scenario, all of them ordered by relevance.
+- XRank / Collapse is not supported.
+- Speller modification is not supported, speller suggestion can be used as normal.
+- Result Template is not supported.
+- Aggregation limitation, if same aggregated field both exist in Sharepoint file types (site, drive, driveItem, list, listItem) and connectors.
+Aggregation result will show two same aggregation buckets with same name, suggest to rename one of its name to bypass the limitation.
+- Specify a signle connection in content source is not supported, there is a plan to change the behavior to allow single connection interleaving search in the future. Please specify only "/external/connections/*" in contentSource.
+
```
Comment (njerigrevious) [inline concepts/search-concept-interleaving.md:132]: These are great limitations - could you please clarify if QueryTemplate is supported or not?

File: concepts/search-concept-interleaving.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+## Known limitations
+
+- Customized Sort is not supported in interleaving scenario, all of them ordered by relevance.
+- XRank / Collapse is not supported.
+- Speller modification is not supported, speller suggestion can be used as normal.
+- Result Template is not supported.
+- Aggregation limitation, if same aggregated field both exist in Sharepoint file types (site, drive, driveItem, list, listItem) and connectors.
+Aggregation result will show two same aggregation buckets with same name, suggest to rename one of its name to bypass the limitation.
+- Specify a signle connection in content source is not supported, there is a plan to change the behavior to allow single connection interleaving search in the future. Please specify only "/external/connections/*" in contentSource.
```
Comment (njerigrevious) [inline concepts/search-concept-interleaving.md:131]: Replace with "Specifying a single connection in content source is not supported, so please only use "/external/connections/*" in contentSource. Note that there is a plan to change the behavior to allow single connection interleaving search in the future."

File: concepts/search-concept-interleaving.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      "from": 0,
+      "size": 25
+    }
+  ]
+}
+```
+
+### Response
+
+The following is an example of the response, which contains one message that matches the search criterion.
```
Comment (njerigrevious) [inline concepts/search-concept-interleaving.md:57]: not sure what is meant by this - should be rephrased to something like "This is an example of a response where each of the specified entitytypes are present in the results"
But is this what you mean here?

**Review by njerigrevious — CHANGES_REQUESTED** [review]

File: concepts/search-concept-interleaving.md (hunk 2 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-interleaving.md]: Done

File: concepts/search-concept-interleaving.md (hunk 1 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-interleaving.md:133]: Clarified, could you please double check?

File: concepts/search-concept-interleaving.md (hunk 3 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-interleaving.md]: Changed, could you please double check the statement.

**Review by njerigrevious — APPROVED** [review]
Thanks for making the updates!

File: api-reference/beta/resources/search-api-overview.md [hunk 4]
```diff
@@ -245,6 +245,7 @@ For backward compatibility, the original properties and types are accessible and
   - [Use search display layout](/graph/search-concept-display-layout)
   - [Search with application permission](/graph/search-concept-searchall)
   - [XRANK search results](/graph/search-concept-xrank)
+  - [search interleaving results](/graph/search-concept-interleaving)
```
Comment (isvargasmsft) [inline api-reference/beta/resources/search-api-overview.md:248]:
```suggestion
  - [Search interleaving results](/graph/search-concept-interleaving)
```

File: concepts/search-concept-interleaving.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+title: "Use the Microsoft Search API to search with interleaved results"
+description: "You can use the Microsoft Search API in Microsoft Graph to search multiple entities in one request and get interleaved results."
+author: "yiwenwang"
+ms.localizationpriority: medium
+ms.prod: "search"
+---
+
+# Supported Entity Combination
+
+| Entity Type |message     | chatMessage| drive       | driveItem  | event      |externalItem | list       | listItem   | person     | site       |
```
Comment (isvargasmsft) [inline concepts/search-concept-interleaving.md:11]: Please move this table to an H2, add the title "Use the Microsoft Search API to search with interleaved results" as an H1 and include a quick introduction to the article.

Comment (isvargasmsft) [reply concepts/search-concept-interleaving.md:11]: In addition, please add a quick introduction (context) to this table.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hi @yiwenwang-ms, 

Thank you very much for your work in this PR. 

Please remove the changelog edits and address some structural suggestions I left. 

Feel free to ping me on the side if you have any questions. 

Kind regards, 

Isaac

**Pull request closed without merging**